docs/SUMMARY.md

```diff
@@ -52,7 +52,7 @@
   * [Read features from the online store](how-to-guides/feast-snowflake-gcp-aws/read-features-from-the-online-store.md)
 * [Running Feast in production](how-to-guides/running-feast-in-production.md)
 * [Upgrading from Feast 0.9](https://docs.google.com/document/u/1/d/1AOsr\_baczuARjCpmZgVd8mCqTF4AZ49OEyU4Cn-uTT0/edit)
-* [Adding a custom provider](how-to-guides/creating-a-custom-provider.md)
```

Comment:
any reason why we removed this? cc @achals @kevjumba

Reply:
oh yeah this looks like a mistake; might have been me

I'll add it in my upcoming docs PR @adchia